### DISCUSSION TITLE
Custom message property

### DISCUSSION THREAD

**Discussion by toudidel** [Koenkk/zigbee2mqtt#10114] [General]
Is it possible to publish message with custom property to topic `/set` and then receive message with that custom property from topic `/get`? Now I can see only officially exposed properties are forwarded.

It'd be very helpful for me to send additional parameters between the scenes.

Example:
I'd like to publish message `{"foo": "bar"}` to device topic, for example `zigbee2mqtt/mydevice/set` and then receive message with set custom property from topic `zigbee2mqtt/mydevice` or `zigbee2mqtt/mydevice/get`?

**Comment by nduhamel** [comment]
I'm looking for the same fonctionality.
I'm using this device https://www.zigbee2mqtt.io/devices/E1744.html#ikea-e1744 which expose brithness messages but i want volume mesage.
Maybe user extension ?

Thank you for your help
